src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5253,4 +5253,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_InDynamicMethodArg" xml:space="preserve">
     <value>Arguments with 'in' modifier cannot be used in dynamically dispatched expessions.</value>
   </data>
+  <data name="IDS_FeatureIndexingMovableFixedBuffers" xml:space="preserve">
+    <value>indexing movable fixed buffers</value>
+  </data>
```

Comment:
I expect a designer file to be updated too. Did you forget to add?

Reply:
I still don't see a designer file.

---
In reply to: [172373788](https://github.com/dotnet/roslyn/pull/24966#discussion_r172373788) [](ancestors = 172373788)

Reply:
reverted and reapplied change. Looks like it has been rebuilt this time. 

---
In reply to: [175236934](https://github.com/dotnet/roslyn/pull/24966#discussion_r175236934) [](ancestors = 175236934,172373788)